examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala

```diff
@@ -174,7 +174,7 @@ object LDAExample {
 
     // Get dataset of document texts
     // One document per line in each text file.
-    val textRDD: RDD[String] = sc.textFile(paths.mkString(","))
+    val textRDD: RDD[String] = sc.textFile(paths.mkString(",")).coalesce(sc.defaultParallelism)
```

Comment:
That seems reasonable, but as this is a basic example, is it necessary or desirable to always introduce a shuffle here? what if the files aren't numerous, or are large?